### ISSUE TITLE
Dependabot alert for transitive dependencies: Minor release

### ISSUE THREAD

**Issue by metters** [hexojs/hexo-deployer-rsync#98]
Hello,

I am receiving dependabot alerts due to the usage of `highlight.js` in `hexo-utils`, which again is used by `hexo-deployer-rsync` in the [latest release 1.0.0](https://github.com/hexojs/hexo-deployer-rsync/blob/1.0.0/package.json#L38):
```
hexo@6.3.0 requires highlight.js@^11.0.1 via a transitive dependency on hexo-util@2.7.0
hexo-deployer-rsync@1.0.0 requires highlight.js@^9.13.1 via a transitive dependency on hexo-util@1.9.1
hexo-renderer-marked@6.0.0 requires highlight.js@^11.0.1 via a transitive dependency on hexo-util@2.7.0
```

Would it be possible for you to trigger a release of a minor version? I have seen that the dependency does not exist in the [current state of the project](https://github.com/hexojs/hexo-deployer-rsync/blob/master/package.json#L33), it just hasn't been released recently.

Kind regards.

**Comment by yoshinorin** [comment]
@metters 
We published a new version `v2.0.0`. It's a major bump but noting breaking changes.

> https://github.com/hexojs/hexo-deployer-rsync/releases/tag/v2.0.0
> https://www.npmjs.com/package/hexo-deployer-rsync/v/2.0.0

Thank you :)

**Comment by metters** [comment]
This is great. Thank you very much!
